app.js

```diff
@@ -8,6 +8,20 @@ function loadMore() {
         hr[i].classList.add('sh');
       }
     document.getElementById('load-more-button').style.display = 'none';
+    document.getElementById('hide-button').style.display = 'block';
+}
+
+function hide() {
+    var paragraphs = document.querySelectorAll('#projects_items .load_more');
+    for (var i = 0; i < paragraphs.length; i++) {
+      paragraphs[i].classList.remove('show');
+    }
+    var hr = document.querySelectorAll('.line_hr');
+    for (var i = 0; i < hr.length; i++) {
```

Comment:
[let's prefer `const` and `let` over `var`](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/)

```suggestion
    const paragraphs = document.querySelectorAll('#projects_items .load_more');
    for (let i = 0; i < paragraphs.length; i++) {
      paragraphs[i].classList.remove('show');
    }
    const hr = document.querySelectorAll('.line_hr');
    for (let i = 0; i < hr.length; i++) {
```